### ISSUE TITLE
Icon and text is misaligned in tabs with Seti UI

### ISSUE THREAD

**Issue by stijnvn** [DanBrooker/file-icons#99]
The Icon and text is misaligned in tabs with Seti UI as can be seen on the screenshot below.

![image](https://cloud.githubusercontent.com/assets/5957239/5549921/f1c4f168-8b96-11e4-80ab-f4d94e485798.png)

**Comment by zephiro** [comment]
+1

![Seti UI](https://cloud.githubusercontent.com/assets/808352/5616871/1d1acce4-94ce-11e4-91bd-ba0f6e44c935.png)

**Comment by ApfelUser** [comment]
+1

**Comment by DanBrooker** [comment]
I think this is an issue with Seti UI, because the icons in tabs are misaligned even when this package is disabled

**Comment by germanocorrea** [comment]
+1

Does have a way to fix it? Create a fork especially for Seti? Something?

**Comment by jsg2021** [comment]
https://github.com/DanBrooker/file-icons/blob/master/styles/icons.less#L145

`top: 3px`

Seti UI has the element position relative so the top 3px pushes it down. In One Dark UI its also misaligned but less severe (the element is not position relative so the top: 3px has no effect)

**Comment by DanBrooker** [comment]
@jsg2021 thanks, I'll take another look at this

**Comment by DanBrooker** [comment]
I haven't figured out a way to fix this without breaking it in other places.

**Comment by s0r00t** [comment]
The same bug occurs on One Dark, using any syntax (here, Seti Syntax) : 
![Screenshot of the bug.](http://i.imgur.com/EfCQ0Al.png)

**Comment by onedr0p** [comment]
+1

**Comment by Alhadis** [comment]
Focussing on fixing this right now.

**Comment by onedr0p** [comment]
Cheers!

**Comment by lenovouser** [comment]
@Alhadis :+1:

**Comment by Alhadis** [comment]
@DanBrooker We could leverage the `.theme-seti-ui` class on the `atom-workspace` element to override the [conflicting properties](https://github.com/jesseweed/seti-ui/blob/master/styles/icons.less). Something like this:
```less
atom-workspace.theme-seti-ui &{
	font-family: name !important;
	padding: 0 !important;
	... etc
}
```
However, that naturally imposes a problem for users who've placed customisations to file-icons in their own stylesheets: they'd need to add `!important;` to each customisation to ensure it overrides everything else.

**Comment by DanBrooker** [comment]
Does `atom-workspace.theme-seti-ui` mean that only Seti UI users would need to `!important` customisations? If so that might be ok and a readme section for Seti UI users would probably be enough. 

But if opening a PR is a cleaner solution for everyone, then that's preferable.

**Comment by Alhadis** [comment]
Actually never mind, I managed to fix it without any `!important` tomfoolery. But it'll require the owner of the Seti UI theme to accept a PR.

Basically, I've wrapped the Seti theme's icon rulesets within a conditional that checks for an existing class in the body. If the class is present, it won't display the icons.

This gives us (and any other icon-related package) a last-minute chance to add `.seti-ui-no-icons` to the `atom-workspace` element, which suppresses a lot of properties that otherwise have the potential for conflict.
